### PR TITLE
Expand user unit tests

### PR DESCRIPTION
**Pull request by roelandkulk**

### CODE REVIEW

File: src/test/java/org/littil/api/user/api/UserResourceTest.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        given()
+                .contentType(ContentType.JSON)
+                .delete("/user/{id}", UUID.randomUUID())
+                .then()
+                .statusCode(404);
+    }
+
+    private UserPostResource getDefaultUser() {
+        UserPostResource user = new UserPostResource();
+        user.setEmailAddress(RandomStringUtils.randomAlphabetic(10) + "@adres.nl");
```
[the remark this answers is not in the document]
Comment (pschildkamp) [reply src/test/java/org/littil/api/user/api/UserResourceTest.java:154]: This goes for multiple tests

**Review by pschildkamp — CHANGES_REQUESTED** [review]
Minor non-trivial comments, but good to do anyway ;). Thanks for your contribution.

File: src/test/java/org/littil/api/user/api/UserResourceTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (roelandkulk) [reply src/test/java/org/littil/api/user/api/UserResourceTest.java]: done

File: src/main/java/org/littil/api/exception/EntityAlreadyExistsExceptionMapper.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package org.littil.api.exception;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+@Slf4j
```
[the remark this answers is not in the document]
Comment (roelandkulk) [reply src/main/java/org/littil/api/exception/EntityAlreadyExistsExceptionMapper.java]: done

**Review by pschildkamp — CHANGES_REQUESTED** [review]

**Pull request merged**